package.json

```diff
@@ -22,7 +22,7 @@
     "Aadit M Shah <aaditmshah@aadit.codes> (https://aadit.codes/)"
   ],
   "scripts": {
-    "format": "prettier --write . && npm run lint -- --fix",
+    "format": "prettier --write . && eslint . --fix",
```

Comment:
How about?

```suggestion
    "format": "prettier --write . && npm run lint:js -- --fix",
```

Reply:
Never mind, I noticed that it's breaking main branch, so exact fix isn't very important :)